docs/book/session.md

```diff
@@ -180,7 +180,8 @@ to seed the session cookie lifetime, if present.
 
 Session containers will typically be passed to your middleware using the
 [SessionMiddleware](middleware.md), via the
-`Mezzio\Session\SessionMiddleware::SESSION_ATTRIBUTE` ("session")
+`Mezzio\Session\SessionMiddleware::SESSION_ATTRIBUTE` ("session") or the
+`Mezzio\Session\SessionInterface::class` ("Mezzio\Session\SessionInterface")
```

Comment:
We need to add notes that attribute `Mezzio\Session\SessionInterface::class` is available since 1.4.0 (future minor release)

Also: do we want deprecate `Mezzio\Session\SessionMiddleware::SESSION_ATTRIBUTE` at the same point?
I think keeping both has no sense, so we should keep just one clear convention.

Reply:
I believe we should keep it at least until the next major number release. It always made sense to me to use fq interface names to store objects into request attributes: it makes it clearer that we are dealing with an object and not a scalar and avoids possible name collision with route-derived attributes. So deprecating it and marking it for removal is fine with me, but that should be decided by you, the laminas team members :-)

Reply:
Keeping both makes sense.

Our API is consistent with the one from psr7-sessions/storageless, which means users can switch between them seamlessly currently by using the "session" attribute name. Ideally, we'll start a trend and we can share a session interface that others will also adopt, and that class name could be used. But for now, allowing either to work makes sense, and I see no need to deprecate.